### PR TITLE
react-week3-homework

### PR DESCRIPTION
**Pull request by humayunadilshahzad**

### CODE REVIEW

**Comment by senner008** [comment]
Good :)
How can you prevent an error when adding a todo to an empty list?

File: react/week3/src/components/todo.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+function Todo({ todo, id, todoDone, todoRemove, todoUpdate }) {
+  console.log(todo);
+
+  const [editedTodo, setEditedtodo] = useState({ description: todo.description, deadline: todo.deadline });
+  const [iscmdEdit, setcmdEdit] = useState(false);
+  //console.log(index, todo.description);
+  // console.log(index,editedTodo.description);
+  return (
+
+    <div className='todo' style={{ textDecoration: todo.isComplete ? 'line-through' : '' }}>
```
Comment (senner008) [inline react/week3/src/components/todo.js:11]: This return part is very hard to read. Try to break it up into smaller properly names parts.

File: react/week3/src/components/todo.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from 'react';
+
+function Todo({ todo, id, todoDone, todoRemove, todoUpdate }) {
+  console.log(todo);
+
+  const [editedTodo, setEditedtodo] = useState({ description: todo.description, deadline: todo.deadline });
+  const [iscmdEdit, setcmdEdit] = useState(false);
+  //console.log(index, todo.description);
```
Comment (senner008) [inline react/week3/src/components/todo.js]: Please remove the console.logs

File: react/week3/src/App.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.todo-list{
```
Comment (senner008) [inline react/week3/src/App.css:40]: I'd prefer not to add styling until you have a solid working version

**Review by senner008 — COMMENTED** [review]
Good 👍
Please look through the suggestions. 
Good work

**Comment by humayunadilshahzad** [comment]
thanx .. yep I knew it ... its got send error when adding new todo at emplty list .. I need to initialize 
id = 1 when list is empty .. 
I noticed ur code to initialize id .. will see it again ..
thanx

**Pull request merged**